armi/physics/fuelCycle/fuelHandlerInterface.py

```diff
@@ -36,6 +36,23 @@ class FuelHandlerInterface(interfaces.Interface):
         :id: I_ARMI_SHUFFLE
         :implements: R_ARMI_SHUFFLE
 
+        This interface allows for a user to define custom shuffle logic that
+        modifies to the core model. Being based on the :py:class:`~armi.interfaces.Interface`
+        class, it has direct access to the current core model.
+
+        User logic is able to be executed from within the
+        :py:meth:`~armi.physics.fuelCycle.fuelHandlerInterface.FuelHandlerInterface.manageFuel` method,
+        which will use the :py:meth:`~armi.physics.fuelCycle.fuelHandlerFactory.fuelHandlerFactory`
+        to search for a Python file specified by the case settings
+        ``fuelHandlerName`` and ``shuffleLogic``. If it exists, the user-specified
```

Comment:
The python file in question can be defined in two separate settings?

Reply:
Good call

https://github.com/terrapower/armi/pull/1589/commits/11d2c6ab0714ea8d436189aef574a11baff9f4ee